thirdeye/thirdeye-frontend/app/pods/rootcause/route.js

```diff
@@ -165,7 +163,7 @@ export default Ember.Route.extend(AuthenticatedRouteMixin, {
     };
 
     let selectedUrns = new Set();
-    let sessionName = 'New Investigation (' + moment().format(dateFormat) + ')';
+    let sessionName = 'New Investigation (' + moment().format(dateFormatFull) + ')';
```

Comment:
this looks like a UI only logic (copy), perhaps this should be handled in the template?

Reply:
I agree it's copy, but it also affects what's stored in the backend. If you come from an anomaly you get a different default than coming from a metric or a stored session.